rbac/internal/views.py

```diff
@@ -1520,6 +1521,54 @@ def lookup_resource(request):
         )
 
 
+def check_relation(request):
+    """POST to check relationship from relations api."""
+    # Parse JSON data from the POST request body
+    req_data = json.loads(request.body)
+
+    # Request parameters for resource lookup on relations api from post request
+    resource_type_name = req_data["resource_type"]["name"]
+    resource_type_namespace = req_data["resource_type"]["namespace"]
+    resource_subject_name = req_data["subject"]["subject"]["type"]["name"]
+    subject_id = req_data["subject"]["subject"]["id"]
+    resource_id = req_data["resource_type"]["id"]
+    resource_relation = req_data["relation"]
+    token = jwt_manager.get_jwt_from_redis()
+
+    try:
+        with create_client_channel(settings.RELATION_API_SERVER) as channel:
+            stub = check_pb2_grpc.KesselCheckServiceStub(channel)
+
+            request_data = check_pb2.CheckRequest(
+                resource=common_pb2.ObjectReference(
+                    type=common_pb2.ObjectType(namespace=resource_type_namespace, name=resource_type_name),
+                    id=resource_id,
+                ),
+                relation=resource_relation,
+                subject=common_pb2.SubjectReference(
+                    subject=common_pb2.ObjectReference(
+                        type=common_pb2.ObjectType(namespace=resource_type_namespace, name=resource_subject_name),
+                        id=subject_id,
+                    ),
+                ),
+            )
+        # Pass JWT token in metadata
+        metadata = [("authorization", f"Bearer {token}")]
+        responses = stub.Check(request_data, metadata=metadata)
+
+        if responses:
```

Comment:
@EvanCasey13 can you please make response json as well as current lookup output ?